src/Runner.Common/ResultsServer.cs

```diff
@@ -94,5 +117,138 @@ public void InitializeResultsClient(Uri uri, string token)
 
             throw new InvalidOperationException("Results client is not initialized.");
         }
+
+        public ValueTask DisposeAsync()
+        {
+            CloseWebSocket(WebSocketCloseStatus.NormalClosure, CancellationToken.None);
+
+            GC.SuppressFinalize(this);
+
+            return ValueTask.CompletedTask;
+        }
+
+        private void InitializeWebsocketClient(string liveConsoleFeedUrl, string accessToken, TimeSpan delay)
+        {
+            if (!string.IsNullOrEmpty(accessToken))
+            {
+                if (!string.IsNullOrEmpty(liveConsoleFeedUrl))
```

Comment:
maybe just check both `if (!string.IsNullOrEmpty(accessToken) && !string.IsNullOrEmpty(liveConsoleFeedUrl))` since we should never land in the method if we don't have that info?

Reply:
I still want to trace a different error message.  Refactored this check.